common/js/src/popup-window/popup-opener.js

```diff
@@ -20,7 +20,7 @@
  * windows.
  */
 
-goog.provide('GoogleSmartCard.PopupWindow.Server');
+goog.provide('GoogleSmartCard.PopupWindow.PopupOpener');
```

Comment:
Maybe we can shorten these names, since there starts to be some duplication in them. What about just "GoogleSmartCard.PopupOpener" and "GoogleSmartCard.InPopupMainScript'?

Reply:
agree